pkg/formatted/color.go

```diff
@@ -58,7 +58,7 @@ func DecorateAttr(attrString, message string) string {
 		return "📨 "
 	case "outputresources":
 		return "📡 "
-	case "steps":
+	case "steps", "check":
```

Comment:
I think u can use `steps` instead of adding new attribute if you are using same `emoji`

Reply:
Added new emoji